src/main/java/dmytro/hadiuchko/springboot/Application.java

```diff
@@ -1,4 +1,4 @@
-package mate.academy.springboot;
+package dmytro.hadiuchko.springboot;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
```

Comment:
Requirement: 'Class with @SpringBootApplication annotation that will have a CommandLineRunner bean'. I don`t see these changes